api/src/database/migrations/20211101A-add-set-null-to-users-created-fields.ts

```diff
@@ -1,26 +1,18 @@
 import { Knex } from 'knex';
 
 export async function up(knex: Knex): Promise<void> {
-	await knex.schema.alterTable('directus_activity', (table) => {
-		table.foreign('user').references('directus_users.id').onDelete('SET NULL');
-	});
-
 	await knex.schema.alterTable('directus_files', (table) => {
 		table.dropForeign('uploaded_by');
 		table.dropForeign('modified_by');
 	});
 
 	return await knex.schema.alterTable('directus_files', (table) => {
-		table.foreign('uploaded_by').references('directus_users.id').onDelete('SET NULL');
-		table.foreign('modified_by').references('directus_users.id').onDelete('SET NULL');
+		table.foreign('uploaded_by').references('directus_users.id').onDelete('NO ACTION').onUpdate('NO ACTION');
+		table.foreign('modified_by').references('directus_users.id').onDelete('NO ACTION').onUpdate('NO ACTION');
```

Comment:
This is already the default right? If true, we can simply remove this whole update 👍🏻